src/vulkan-renderer/render_graph.cpp

```diff
@@ -486,30 +505,26 @@ void RenderGraph::compile(const RenderResource *target) {
         }
     }
 
-    m_log->trace("Allocating command buffers for stage:");
+    m_log->trace("Creating semaphore for stage:");
 
     // Allocate command buffers and finished semaphore.
     for (const auto *stage : m_stage_stack) {
         auto &physical = *stage->m_physical;
-        physical.m_finished_semaphore =
-            std::make_unique<wrapper::Semaphore>(m_device, "Finished semaphore for stage " + stage->m_name);
+        physical.m_finished_semaphore = std::make_unique<wrapper::Semaphore>(m_device, stage->m_name);
```

Comment:
Pretty sure this can be removed now that it's unused.

Reply:
What about `return m_stage_stack.back()->m_physical->m_finished_semaphore->get();` in `RenderGraph::render`?